ci/travis-install.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 set -ev
 
-if [ "$TRAVIS_NODE_VERSION" == "v0.12.17" ]; then 
-    npm install --force && npm link && npm link webpack;
-else 
-    npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
-fi
+npm install yarn -g && yarn install && yarn link || true && yarn link webpack;
```

Comment:
Probably correct. But I wanted to make as minimal CI changes as possible for now. We'll likely refactor this out, or leave since we do not include the `yarn.lock` file.